packages/components/src/internal/components/editable/models.ts

```diff
@@ -306,24 +312,23 @@ export class EditorModel
             if (renderer?.getEditableRawValue) {
                 row = row.set(col.name, renderer.getEditableRawValue(values));
             } else if (col.isLookup()) {
-                if (col.isExpInput() || col.isAliquotParent()) {
+                if (col.isAliquotParent()) {
```

Comment:
As we talked in chat, the use of rowId resulted in broken parent name expression usage.

Reply:
I was able to resolve this issue by reverting my change here, but also by adding some similar special casing to the getUpdatedData method that checks col.isExpInput(), and if so compares against the displayValue instead of value.